sklearn/neural_network/_stochastic_optimizers.py

```diff
@@ -0,0 +1,259 @@
+"""Stochastic optimization methods for MLP
+"""
+
+# Authors:  Jiyuan Qian <jq401@nyu.edu>
+# Licence: BSD 3 clause
+
+import numpy as np
+
+
+class BaseOptimizer(object):
```

Comment:
I guess this can also be considered fixed as I have removed `_get_updates` from this base class